.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: go
+
+os: 
+  - linux
+  - osx
+
+branches:
+  only: 
+    - master
```

Comment:
The default branch is `main` and not `master`

Reply:
oops 😥

Reply:
Fixed!